### PR TITLE
🔨 remove old Frontpage code

### PR DESCRIPTION
**Pull request by ikesau**
Grepped each of the classes in `homepage.scss` and removed them if they were only used in `Frontpage.tsx`

Also removes the old `frontPageConfig` stuff.

### CODE REVIEW

File: baker/siteRenderers.tsx [hunk 1]
```diff
@@ -256,103 +253,27 @@ export const renderPost = async (
...
-    const gdocHomepageResult = await db.knexRawFirst<{ id: string }>(
-        `SELECT id FROM posts_gdocs WHERE content->>"$.type" = "${OwidGdocType.Homepage}" AND published = TRUE`,
-        db.knexInstance()
+    const gdocHomepageResult = await knex.raw<{ id: string }>(
+        `--sql
+        SELECT id
+        FROM posts_gdocs
+        WHERE content->>"$.type" = "${OwidGdocType.Homepage}"
+        AND published = TRUE`
     )
```
Comment (mlbrgl) [inline baker/siteRenderers.tsx:263]: We probably need to revert this change:
- the "--" needs a space after it (otherwise I get a syntax error)
- gdocHomepageResult is an array so gdocHomepageResult.id below doesn't return what we would expect

**Review by mlbrgl — CHANGES_REQUESTED** [review]

**Comment by mlbrgl** [comment]
The removal looks ok and makes sense, but I'm not too sure about the new Homepage.scss. I'm assuming it would mostly disappear on rebase?

File: baker/siteRenderers.tsx (hunk 1 above)
Comment (ikesau) [reply baker/siteRenderers.tsx]: Wow yeah this was completely broken. Rewrote it and put the query in `db.ts`

**Comment by ikesau** [comment]
@mlbrgl To be clear, I renamed `css/homepage.scss` to `gdocs/pages/Homepage.scss`.

Everything in it is now used. There's just some rules from the old page that are still used (the newsletter stuff) which follows a slightly different naming convention.
